docs/en/observability/synthetics-params-secrets.asciidoc

```diff
@@ -131,9 +136,14 @@ As an example, the syntax `${URL}` for instance, to reference a variable named `
   params:
     url: ${URL}
   source:
-    zip_url:
-      url: "https://github.com/elastic/synthetics-demo/archive/refs/heads/main.zip"
-      folder: "synthetics-tests"
+    inline:
+      script: |-
+      step("load homepage", async () => {
+        await page.goto(params.url);
+      });
+      step("hover over products menu", async () => {
+        await page.hover('css=[data-nav-item=products]');
+      });
 ----
 
 To setup the {heartbeat} keystore see the {heartbeat-ref}/keystore.html[{heartbeat} keystore documentation]. 
```

Comment:
I'm thinking we should remove references to the heartbeat keystore as well, and focus on using params instead.